### PR TITLE
Change instructions so users will be added by creating an issue

### PR DESCRIPTION
**Pull request by brianamarie**
## Overview
**TL;DR**

This changes the instructions so users will be added by creating an issue rather than commenting on an existing issue. 

This will need to be merged at the same time as [this related pull request in another repository having to do with managing the webhook](https://github.com/githubteacher/practice-sinatra/pull/4).

This solves several problems: 
- Users have been creating issues instead of commenting by mistake
- Threads can only have 2500 comments, which we have maxed out

### Next Steps
- [ ] Test and get 👍 

### Review

@github/services-training How does this look? Is this a direction we want to go?

### CODE REVIEW

**Comment by hectorsector** [comment]
🚢-ing thanks to @stoe's fantastic work in https://github.com/githubteacher/practice-sinatra/pull/4

🎉
